modules/plugin/imagemosaic/src/main/java/org/geotools/gce/imagemosaic/ImageMosaicFormat.java

```diff
@@ -190,6 +190,18 @@
                     new Boolean[] {Boolean.TRUE, Boolean.FALSE},
                     Boolean.FALSE);
 
+    /**
+     * A multi-crs reader supporting alternative CRSs can produce output to one of them when the
+     * requested grid geometry is expressed in that CRS and this flag is set to true. No
+     * reprojections will be performed on granules having native CRS matching the target one.
```

Comment:
"When this read parameter is set to true, the reader will produce output in the requested CRS, assuming it matches one of the native CRSs, as reported by the MULTICRS_EPSGCODES metadata entry, and that the requested grid geometry is expressed in said CRS. When set to false (default), then only the native CRS declared by GridCoverage2DReader.getCoordinateReferenceSystem will be produced in output".

(I skipped adding the links)

Reply:
Updated accordingly